### PR TITLE
new-changes-in-plans-description

### PR DESCRIPTION
**Pull request by etf-yrm**
added changes in description plans

### CODE REVIEW

File: pricing.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
 <div class="inner_top"><span class="t_hm">pilot</span><span class="t_h0">$99</span><span class="t_h2">Per month.</span><span class="t_h2 t_h2_after">Up to 20 devices.<br/></span></div>
 <div class="inner_middle"><div class="vertical_aligner">
   <ul>
+	<li>Single user only</li>
```
Comment (vpetersson) [inline pricing.html]: Indentation error.

File: pricing.html [hunk 2]
```diff
@@ -32,6 +33,7 @@
 <div class="inner_top"><span class="t_hm">standard</span><span class="t_h0">$299</span><span class="t_h2">Per month.</span><span class="t_h2 t_h2_after">Up to 50 devices.</span></div>
 <div class="inner_middle"><div class="vertical_aligner"><span class="descs"><span class="t_h2">All Pilot Features</span></span>
   <ul>
+  	<li>3 users included</li>
```
Comment (vpetersson) [inline pricing.html]: Indentation error.

**Comment by vpetersson** [comment]
@etf-yrm make sure to always use soft tabs.

File: pricing.html (hunk 1 above)
Comment (etf-yrm) [reply pricing.html]: @vpetersson fixed this in last commit

File: pricing.html (hunk 2 above)
Comment (etf-yrm) [reply pricing.html]: @vpetersson fixed this in last commit

**Pull request merged**